pkg/types/validation/installconfig_test.go

```diff
@@ -1277,28 +1277,6 @@ func TestValidateInstallConfig(t *testing.T) {
 			}(),
 			expectedError: `Invalid value: "IPv6": single-stack IPv6 is not supported for this platform`,
 		},
-		{
```

Comment:
Don't these tests still apply? It is still possible to select `OpenShiftSDN` as the network type, it is just not the default.

Reply:
good catch. I added them back.